actors/builtin/miner/miner_actor.go

```diff
@@ -330,7 +334,7 @@ type PoStPartition = miner0.PoStPartition
 //	// The ticket randomness on the chain at the chain commit epoch.
 //	ChainCommitRand abi.Randomness
 //}
-type SubmitWindowedPoStParams = miner0.SubmitWindowedPoStParams
+type SubmitWindowedPoStParams = minertypes.SubmitWindowedPoStParams
```

Comment:
Do these param substitutions need to happen here in the same way the proof types change needs to happen?

Reply:
No, they don't -- i think. Undoing.